.github/workflows/post-pr-cleanup.yml

```diff
@@ -0,0 +1,61 @@
+name: cleanup-codecov-reports
+
+
+# This workflow deletes the html reports for the given PR to tidy up the helper repository
+
+on:
+  pull_request:
+    types: [closed]
+    branches: [main]

```

Comment:
Sollten wir das nicht auch bei anderen branches (außer stable) machen? Wenn ich jetzt von einem X1 branch in X2 merge und den X1 schließe, bleibt die codecov da...

Reply:
Die allgemeine Pipeline läuft auch nur bei PRs gegen stable und main..und bei stable kommt der PR ja aus dem Master, da will ich die codecov nicht löschen